src/Lecture3.hs

```diff
@@ -49,15 +53,18 @@ data Weekday
     | Friday
     | Saturday
     | Sunday
-    deriving (Show, Eq)
+    deriving (Show, Eq, Enum, Ord )
 
 {- | Write a function that will display only the first three letters
 of a weekday.
 
 >>> toShortString Monday
 "Mon"
 -}
-toShortString = error "TODO"
+
+toShortString :: Weekday -> String
+toShortString w = take 3 (show w)
```

Comment:
Absolutely correct implementation 👍🏻 
You can use function composition and eta-reduce it:
```suggestion
toShortString = take 3 . show
```